packages/legend-query-builder/src/stores/filter/QueryBuilderFilterOperator.ts

```diff
@@ -27,7 +27,7 @@ import type {
 export abstract class QueryBuilderFilterOperator implements Hashable {
   readonly uuid = uuid();
 
-  abstract getLabel(filterConditionState: FilterConditionState): string;
+  abstract getLabel(): string;
```

Comment:
This param was not used in any of the implementations of this function, so I removed it.